src/app/node-data/style.scss

```diff
@@ -17,6 +17,10 @@ div.km-os-options {
   margin: 0 0 10px;
 }
 
+mat-checkbox.km-use-autoscaling {
```

Comment:
s/km-use-autoscaling/use-autoscaling

We tend to use `km` prefix only for global classes.